src/account-balance/agent.js

```diff
@@ -47,13 +47,23 @@ function createAlert(
   return Finding.fromObject(findingObject);
 }
 
+const validateConfig = (config) => {
+  let ok = false;
+  let errMsg = "";
+
+  if (config["accountBalance"] === undefined) {
+    errMsg = "accountBalance not provided!";
+    return { ok, errMsg };
+  }
+
```

Comment:
Consider adding checks here for the key/value pairs in the Objects stored under the `accountBalance` key:
```suggestion
  const accountBalance = config;
  const contractNames = Object.keys(accountBalance);
  for (let i=0; i < contractNames.length; i++) {
    const { address, thresholdEth, alert: { type, severity } } = accountBalance[contractNames[i]];
    
    // verify that the address is a valid 20-byte hex string
    if (!ethers.utils.isHexString(address, 20)) {
        errMsg = `address not valid 20-byte hex string: ${address}`;
        return { ok, errMsg };
    }
    
    // verify that the threshold specified can be converted to an ethers.BigNumber
    try {
      ethers.BigNumber.from(thresholdEth);
    } catch (error) {
      errMsg = `Cannot convert value in thresholdEth to ethers.BigNumber: ${thresholdEth}`;
      return { ok, errMsg };
    }
    
    // verify that the finding type is valid
    if (!FindingType[type]) {
      errMsg = `Invalid Finding Type specified: ${type}`;
      return { ok, errMsg };
    }
    
    // verify that the finding severity is valid
    if (!FindingSeverity[severity]) {
      errMsg = `Invalid Finding Severity specified: ${severity}`;
      return { ok, errMsg };
    }
  }
```